src/sections/Landscape/LandscapeGrid.style.js

```diff
@@ -407,6 +407,7 @@ export const LandscapePageWrapper=styled.div`
 	.react-tabs__tab {
 		bottom: 0px;
 		padding: 1.25rem;
+		background-color: #5b8b7e;
```

Comment:
Could you please use a color from our theme and not hard-code the colors

Reply:
Ya sure but can you tell me where can I find the theme colors so that I can choose a suitable color for that?

Reply:
There's a file called themeStyles.js where you'd be able to find the colors, you could also take look at https://layer5.io/brand

Reply:
Can you check it now I have used predefined colors?
New change:
![Screenshot from 2022-01-18 00-39-03](https://user-images.githubusercontent.com/88609894/149825560-48fb5cda-5b11-4663-88b5-67830aa8a380.png)

Reply:
The problem here is that the same challenge of users not being shown the outline of a tab (so that they realize that this is a multi-tabbed table still remains.

Reply:
Ok understood.. So I should give a good outline for each tab so that users can understand... But can you suggest a good color of the outline that can be matched with the UI?...Or I can do some work on the outline and show you so that you can give the suggestion?..what do you think?

Reply:
![Screenshot from 2022-01-18 09-31-24](https://user-images.githubusercontent.com/88609894/149868851-14cc9c1f-a43d-467f-a12a-641efe25a341.png)

Is it looking good now?..I have not pushed the latest changes yet..before that can you check it?

Reply:
Yup this looks good.

Reply:
Ok..so let me push the latest change